drivers/can/Kconfig.stm32

```diff
@@ -9,9 +9,10 @@
 config CAN_STM32
 	bool "STM32 CAN Driver"
 	depends on SOC_FAMILY_STM32
+	default y
 	select USE_STM32_HAL_CAN
 	help
-	  Enable STM32 CAN Driver (tested on stm32F0 series)
+	  Enable STM32 CAN Driver
```

Comment:
@alexanderwachter Why did you close the PR? We could use it to update help section.

Reply:
I'll open a new one.